plugins/myst/package.json

```diff
@@ -22,6 +23,7 @@
   "dependencies": {
     "@stencila/plugin": "^2.0.0-alpha.1",
     "@stencila/types": "^2.0.0-alpha.25",
+    "mdast": "^3.0.0",
```

Comment:
We also have `myst-spec` and `myst-spec-ext` which is our staging ground for types. Those two libraries are the main ones that are defining our mdast-extended types.